charts/azure-service-operator/Chart.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: v2
 name: azure-service-operator
-version: 1.5.0
+version: 1.6.0
 appVersion: 1.0.24492
 description: Deploy components and dependencies of azure-service-operator
 home: https://github.com/Azure/azure-service-operator
 sources:
   - https://github.com/Azure/azure-service-operator
 dependencies:
   - name: aad-pod-identity
-    version: 1.5.5
+    version: 4.1.1
```

Comment:
This is a massive version bump!

Reply:
It gave me a fright at first as well but it's not quite as bad as it sounds. The aad-pod-identity chart version used to be in lock-step with the underlying package version but they floated it at some point - the actual aad-pod-identity version is going from 1.5.5 to 1.8.0.